### PR TITLE
AFK, Respawn and Game tab tweaks

### PR DESCRIPTION
**Pull request by Rohesie**
Nothing happened here.

### CODE REVIEW

File: code/__DEFINES/mobs.dm [hunk 1]
```diff
@@ -313,6 +313,11 @@ GLOBAL_LIST_INIT(xenoupgradetiers, list(XENO_UPGRADE_BASETYPE, XENO_UPGRADE_INVA
 #define IS_MOTH (1<<7)
 //=================================================
 
+//AFK status
+#define MOB_CONNECTED 0
+#define MOB_AFK 1 //Still within the grace period.
```
Comment (psykzz) [inline code/__DEFINES/mobs.dm]: AFK isn't AFK here, its disconnected but within grace.

I would personally find another name as its confusing. To me AFK is still connected, but not moved for X time.
However from your implementation it looks like its disconnected within 5/15 mins.

Comment (Rohesie) [reply code/__DEFINES/mobs.dm]: Which name would you give it?

Comment (Rohesie) [reply code/__DEFINES/mobs.dm]: Changed it to `MOB_RECENTLY_DISCONNECTED`

Comment (psykzz) [reply code/__DEFINES/mobs.dm]: Sorry i didn't have a name at the time, `DISCONNECTING` would also work

**Pull request merged**